src/main/java/mate/academy/hw4/myArrayList/SimpleArray.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.hw4.myArrayList;
+
+import java.util.Iterator;
+
+public class SimpleArray<E> implements Simple<E> {
+    private E[] values;
+
+    public SimpleArray() {
+        values = (E[]) new Object[0];
```

Comment:
Why zero initial size ?

Reply:
In the constructor, I initialized the value, the array should be empty